lib/sales_engine.rb

```diff
@@ -9,9 +9,9 @@ class SalesEngine
                 :invoices
 
   def initialize(csv_files)
-    @items = ItemRepo.new(csv_files[:items], self)
-    @merchants = MerchantRepo.new(csv_files[:merchants], self)
-    @invoices = InvoiceRepo.new(csv_files[:invoices], self)
+    @items = ItemRepo.new(csv_files[:items], self) unless csv_files[:items] == nil
```

Comment:
I like this method for cutting down out run time and accessing the different repos separate instead of all at one time.

Reply:
Amazing job and research on the cut time of this method and testing!!!

Reply:
the use of unless is exactly what we needed here.